packages/FormElement/src/FormElement.js

```diff
@@ -122,59 +45,29 @@ FormElement.types = {
 };
 
 const propTypes = {
-  children: PropTypes.node.isRequired,
-
-  /** Should show is optional text besides the label or not. Will not show if hasRequiredLabel prop is true */
-  hasOptionalLabel: PropTypes.bool,
-
-  /** Should show is required text besides the label or not. Takes presendence over hasOptionalLabel prop */
-  hasRequiredLabel: PropTypes.bool,
-
-  /** ID for the child element. */
   id: PropTypes.string,
 
+  children: PropTypes.node.isRequired,
+
   /** Should be disabled or not, default is false. */
   isDisabled: PropTypes.bool,
 
-  /** Should label and children be inline or not, default is false. */
-  isInline: PropTypes.bool,
-
-  /** Should label be hidden, default is false. Note: this is discouraged because of accessibility requirements. */
-  isLabelVisuallyHidden: PropTypes.bool,
-
-  /** Label text of this field. */
-  label: PropTypes.node.isRequired,
-
   /** Size of the label, error, help and description (font size, min-height, padding, etc). */
   size: PropTypes.oneOf([FormElement.types.size.SMALL, FormElement.types.size.MEDIUM, FormElement.types.size.LARGE]),
 
   /** FormElement contains multiple children so Renders a legend element instead of label. */
   hasFieldSet: PropTypes.bool,
-
-  onClickLabel: PropTypes.func,
 };
 
 const defaultProps = {
-  hasOptionalLabel: false,
-  hasRequiredLabel: false,
-  id: null,
+  hasFieldSet: false,
+  id: "",
   isDisabled: false,
-  isInline: false,
-  isLabelVisuallyHidden: false,
   size: FormElement.types.size.MEDIUM,
-  hasFieldSet: false,
-  onClickLabel: () => {},
 };
 
 FormElement.displayName = "FormElement";
 FormElement.propTypes = propTypes;
 FormElement.defaultProps = defaultProps;
 
-FormElement.subComponentDisplayNames = subComponentDisplayNames;
-FormElement.Content = Content;
-FormElement.Instructions = Instructions;
-FormElement.Description = Description;
-FormElement.Error = ErrorMessage;
-FormElement.Help = Help;
```

Comment:
I prefer this style of packaging up the component.  I see it is how you are packaging the `Fieldset` and I think they should be consistent.  It means you can't import the subcomponents separately on the same line, but if you really want to, you could destructure the `FormElement` (or `Fieldset`) to clean up the JSX.  

If this is confusing, let's discuss.  I tried it out locally, and I think it's a bit clearer.

Reply:
Ok i've switched it